xml/System/Object.xml

```diff
@@ -420,15 +420,17 @@
  If <xref:System.Object.Finalize%2A> or an override of <xref:System.Object.Finalize%2A> throws an exception, and the runtime is not hosted by an application that overrides the default policy, the runtime terminates the process and no active `try`/`finally` blocks or finalizers are executed. This behavior ensures process integrity if the finalizer cannot free or destroy resources.  
   
 <a name="Notes"></a>   
-## Notes for implementers  
- You should override <xref:System.Object.Finalize%2A> for a class that uses unmanaged resources such as file handles or database connections that must be released when the managed object that uses them is discarded during garbage collection.  
+## Overriding the Finalize method 
+ You should override <xref:System.Object.Finalize%2A> for a class that uses unmanaged resources such as file handles or database connections that must be released when the managed object that uses them is discarded during garbage collection. You should not implement a <xref:System.Object.Finalize%2A> method for managed objects, bevause the garbage collector releases managed resources automatically.  
   
 > [!IMPORTANT]
 >  If a <xref:System.Runtime.InteropServices.SafeHandle> object is available that wraps your unmanaged resource, the recommended alternative is to implement the dispose pattern with a safe handle and not override <xref:System.Object.Finalize%2A>. For more information, see [The SafeHandle alternative](#SafeHandle) section.  
   
  The <xref:System.Object.Finalize%2A?displayProperty=fullName> method does nothing by default, but you should override <xref:System.Object.Finalize%2A> only if necessary, and only to release unmanaged resources. Reclaiming memory tends to take much longer if a finalization operation runs, because it requires at least two garbage collections. In addition, you should override the <xref:System.Object.Finalize%2A> method for reference types only. The common language runtime only finalizes reference types. It ignores finalizers on value types.  
+
+The scope of the <xref:System.Object.Finalize?displayProperty=fullName> method is `protected`. You should maintain this limited scope when you override the method in your class. By keeping a <xref:System.Object.Finalize%2A> method protected, you prevent users of your application from calling an object's <xref:System.Object.Finalize%2A> method directly.
```

Comment:
> \<xref:System.Object.Finalize?displayProperty=fullName\>

Did you intend to leave the `%2A` off? ... It renders with parentheses without it `System.Object.Finalize()`.